README.md

```diff
@@ -177,6 +177,8 @@ to detect corruptions that occurred during storage. It is therefore recommended
 lz4 compression by means of the ```--disable-lz4``` configure flag.
 
 Detailed tracing will be enabled when the environment variable `LIBRAFT_TRACE` is set upon startup.
+The value of it can be in `0..8` range and reperesents a tracing level, where
+`0` means "no traces" emitted, and `8` enables maximum verbosity.
```

Comment:
I'm used to `0..8` denoting the range `0 <= x < 8`, could this spell it out unambiguously?

Reply:
Agree, `[0, 8]` seems to look better.